cmd/rbacgen/main.go

```diff
@@ -0,0 +1,93 @@
+package main
```

Comment:
Please create README in this package which describing purpose of this tool and its flags.